### ISSUE TITLE
ffmpeg binary

### ISSUE THREAD

**Issue by Asp3ctus** [jyapayne/Web2Executable#162]
in readme it said : Working media out of the box (sound and video)
but i had to download binary https://github.com/iteufel/nwjs-ffmpeg-prebuilt/releases
to get mp4 support in html5

**Comment by jyapayne** [comment]
Right. It should say "working open source audio and video". The formats supported are only the open source formats like ogg and matroska. The reason is so that if you want to sell what you bundle, you aren't restricted under a license. Note that if you use FFMPEG, you are restricted under the GPL license and cannot legally sell what you bundle.

**Comment by Asp3ctus** [comment]
Well i guess it should be a setting then

**Comment by jyapayne** [comment]
You're correct. It should be a setting to download from an external source.
